src/test/java/com/epam/tc/hw3/ex1/IndexPageData.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.tc.hw3.ex1;
+
+import java.util.List;
+
+public class IndexPageData {
+
+    public static String PROPERTIES_PATH = "src/test/resources/hw3/test.properties";
```

Comment:
Upper case names uses only for final constants. Actually all data in such data classes can be final. (Valid for string, list data)

Reply:
Data in the data classes were changed to final, PROPERTY_PATH moved to PropertyReader